app/javascript/hooks/mutations/admin/site_settings/useDeleteSiteSetting.jsx

```diff
@@ -0,0 +1,20 @@
+import { useMutation, useQueryClient } from 'react-query';
+import { toast } from 'react-hot-toast';
+import axios from '../../../../helpers/Axios';
+
+export default function useDeleteSiteSetting(name) {
+  const queryClient = useQueryClient();
+
+  return useMutation(
+    (data) => axios.delete(`/admin/site_settings/${name}.json`, data),
+    {
+      onSuccess: () => {
+        queryClient.invalidateQueries('getSiteSettings');
+        toast.success('Branding Image deleted');
```

Comment:
Is this always gonna be a `Branding Image deleted`?
Shouldn't this be something generic `Site setting deleted`?